base/tests.py

```diff
@@ -219,7 +219,7 @@ class TestGeo(TestCase):
                                       'southwest': {'lat': -0.7461185, 'lng': 8.753189299999999}},
                            'viewport': {'northeast': {'lat': -0.7244695, 'lng': 8.767470699999999},
                                         'southwest': {'lat': -0.7461185, 'lng': 8.753189299999999}},
-                           'location': {'lat': -0.7345885999999999, 'lng': 8.760486199999999},
```

Comment:
Non capisco -- da dove vengono questi nuovi valori? Se i valori cambiano leggermente in modo periodico, e' il caso che il test venga effettuati in modo da non rompersi ogni volta, magari con un controllo fino alla N^a cifra decimale.